easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -24,7 +24,7 @@ prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 
 # make sure that system libraries are also considered by ld and ld.gold is also built
 # --enable-plugins should be used whenever --enable-gold is used, see http://llvm.org/docs/GoldPlugin.html
-configopts = '--with-sysroot=/ --enable-gold --enable-ld=default --enable-plugins'
+configopts = '--with-sysroot=/ --enable-gold --enable-ld=default --enable-plugins --enable-shared --enable-static'
```

Comment:
Let's not do it on this one. It's special. It should be minimal and it's only purpose should be to build `GCC`. Preferably, nothing links dynamically to it.